templates/patterns/page_header/page_header.ui_patterns.yml

```diff
@@ -5,20 +5,20 @@ page_header:
     identity:
       type: "text"
       label: "Identity"
-      description: "Name of the site"
+      description: "Name of the site (deprecated)"
```

Comment:
Is this deprecated? We should double check it with @kugta

Reply:
Is not deprecated (I will change that) and one ticket is already created to follow this part.